arrow/server.py

```diff
@@ -18,15 +19,24 @@
 
 @app.route('/avroToRawls', methods={'POST'})
 async def avro_to_rawls(request):
-    url = request.json['url']
-    defaults = {'b64-decode-enums': True, 'prefix-object-ids': True}
-    request_options = request.json.get('options', {})
-    options = {**defaults, **request_options}
-
-    avro = urllib.request.urlopen(url)
-    reader = fastavro.reader(avro)
-    result = await translate(reader, options)
-    return response.json(result)
+    try:
+        try:
```

Comment:
nit/TOL: I think it's generally considered best practice for readability to not have nested try/catch's and instead move internal try/catches to their own methods and call those from within the larger try/catch. This method is relatively few lines/not super complex though so if that feels like overkill it's certainly not a necessary change just a thought

Reply:
Discussed as mob. Going to investigate extracting outer try into a decorator.

Reply:
Got decorator working. Please re-review. Thanks!

Reply:
Looks good to me, my concern is addressed thanks!

Reply:
Absolutely! It was good feedback that led to overall better code because the same error handling was easy to apply to other endpoints.